.dockerignore

```diff
@@ -0,0 +1,5 @@
+Dockerfile
+.travis.yml
+.gitignore
+demo/etc
+tox.ini
```

Comment:
This line seems out of place

Reply:
You mean ``tox.ini``, tests are not run in the Docker image. I guess we could include the tox configuration of course, but I did not find it useful.

Reply:
Alright then